apps/console/src/features/remote-repository-configuration/api/remote-repo-config.ts

```diff
@@ -52,15 +53,15 @@ export const getRemoteRepoConfigList = (): Promise<AxiosResponse<InterfaceRemote
         });
 }
 
-export const getRemoteRepoConfig = (id: string): Promise<AxiosResponse> => {
+export const getRemoteRepoConfig = (id: string): Promise<AxiosResponse<InterfaceRemoteConfigDetails>> => {
```

Comment:
Shall we rename `InterfaceRemoteConfigDetails` to `RemoteConfigDetailsInterface`?

Reply:
Fixed with commit 20f0488